src/mlpack/core/dists/discrete_distribution.hpp

```diff
@@ -219,6 +244,25 @@ class DiscreteDistribution
   std::vector<arma::vec> probabilities;
 };
 
+/**
+ * Calculates the Discrete log probability function for each
```

Comment:
Please capitalize the `log` here.

Reply:
I don't know about this one, I would actually suggest 'discrete log-probability function'.